src/index.c

```diff
@@ -1158,6 +1158,11 @@ static int OI_SkipTo(void *ctx, t_docId docId, RSIndexResult **hit) {
     // reset current pointer since this might have been a prior
     // virt return
     nc->base.current = nc->child->current;
+    if(!nc->base.current){
```

Comment:
@mnunberg make sure to check out this fix as it prevents the crash but not sure about its correctness

Reply:
I'll look into it later today; for the time being, remove this fix and I'll check it out later today. It seems like something that can happen on 1.6 as well; though it is very much an edge case.